cypress/e2e/felony/class-a/pages/f-classa-main-1-ineligble.cy.ts

```diff
@@ -0,0 +1,5 @@
+import testIneligiblePage from "../../utils/testIneligiblePage"
+
+describe('f-classa-main-1-ineligble', () => {
+  testIneligiblePage('f-classa-main-1-ineligble');
```

Comment:
I really liked this idea! I will use this on fishing pages that have same tests to not duplicate too much! :tada: 

Im just wondering if we really need use this in this situation, do you know/think we will use that `testIneligiblePage()` in any other page?

I took a look in figma and i could not find any other place to use this. Am i missing something?

If we dont have any other place to use this function, would be better to just put our test code inside here.

Reply:
@LeoFuna 
Yes, there are several endpoints that lead to an ineligible page that share identical content. You can find the link [here](https://www.figma.com/file/icusYj7kJl2xmYkudIZEG4/CV-Current-Design-Files?type=design&node-id=19262-109354&mode=design&t=idQIfnn7xjzNXK0R-4).
I recommend zooming out to view the continuation branches for a better understanding. For example: Clicking 'Yes' on `f-start-classb-1-cont` will direct users to `f-offense-classbpro-1-main`. There are four continuation branches that require zooming out to see.

Reply:
Perfect!